test/e2e/helpers.js

```diff
@@ -555,24 +555,6 @@ const multipleGanacheOptionsForType2Transactions = {
   hardfork: 'london',
 };
 
-const generateGanacheOptions = ({
-  secretKey = PRIVATE_KEY,
-  balance = convertETHToHexGwei(DEFAULT_GANACHE_ETH_BALANCE_DEC),
-  ...otherProps
-}) => {
-  const accounts = [
-    {
-      secretKey,
-      balance,
-    },
-  ];
-
-  return {
-    accounts,
-    ...otherProps, // eg: hardfork
-  };
-};
-
```

Comment:
this method is not needed, as you can pass any option in the `localNodeOptions` variable in the test suite, direclty, without any method. The default accounts are already set using the mnemonic at the node class level